src/parsec.rs

```diff
@@ -136,22 +208,20 @@ impl ParsecMap {
     }
 
     pub fn gossip_recipients(&self) -> Vec<&id::PublicId> {
-        self.map
-            .values()
+        self.get_parsec_values()
             .last()
-            .map(|parsec| parsec.gossip_recipients().collect())
+            .map(|p| p.gossip_recipients().collect())
```

Comment:
prefer remain with the `parsec` ?

Reply:
Yes good catch! That one actually slipped through by mistake